<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1204</width>
    <height>805</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="autoLlenarBoton">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>260</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 255, 0);</string>
    </property>
    <property name="text">
     <string>Auto llenar</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Poblacion inicial</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Numero de paquetes</string>
    </property>
   </widget>
   <widget class="QPushButton" name="CalcularBoton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>260</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(14, 113, 179);</string>
    </property>
    <property name="text">
     <string>Calcular</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="poblacionInicialText">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>40</y>
      <width>191</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(198, 0, 0);
 border: 2px solid</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="numeroPaquetes">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>160</y>
      <width>181</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(198, 0, 0);
 border: 2px solid</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Poblacion maxima</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="poblacionMaximaText">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>100</y>
      <width>181</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(198, 0, 0);
 border: 2px solid</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>40</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Decendencia</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>80</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Mutacion individuo</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>120</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Mutacion gen</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Tamaño contenedor</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>10</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="decendenciaText">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>40</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(198, 0, 0);
 border: 2px solid</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="mutacionIndividuoText">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>80</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(198, 0, 0);
 border: 2px solid</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="mutacionGenText">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>120</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(198, 0, 0);
 border: 2px solid</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tamanoContenedor">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>200</y>
      <width>181</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(198, 0, 0);
 border: 2px solid</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>325</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Paquetes Inicial</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="paquetesInicialLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>370</y>
      <width>291</width>
      <height>381</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(198, 0, 0);
 border: 2px solid;</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="paquetesFinalLabel">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>320</y>
      <width>801</width>
      <height>431</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(198, 0, 0);
 border: 2px solid;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>276</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Paquetes final</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>40</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Generaciones</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="espacioText">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>210</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(198, 0, 0);
 border: 2px solid</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="costoText">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>210</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(198, 0, 0);
 border: 2px solid</string>
    </property>
   </widget>
   <widget class="QPushButton" name="anadirBoton">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>210</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 255, 0);</string>
    </property>
    <property name="text">
     <string>Añadir</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>176</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Espacio</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>176</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Costo</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="generacionesText">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>40</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(198, 0, 0);
 border: 2px solid</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1204</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
